### DISCUSSION TITLE
Browser compatibility issue with glTF models

### DISCUSSION THREAD

**Discussion by davidmaruscsak00** [cables-gl/cables_docs#229] [Answers & Questions] [answered]
Hello,

I have designed three scenes. Each is a miniature city. 

I have exported them as .glb files from Blender.

The scenes are working fine for me in Firefox and Safari, however in Chrome, the vertex of the glTF models are quite chaotic.

Here you can find the scenes:
scene1:
https://cables.gl/p/ynJzT_

scene2:
https://cables.gl/p/xOueX_

scene3:
https://cables.gl/p/1sDh0-

<img width="1197" alt="Screenshot 2022-04-24 at 20 04 34" src="https://user-images.githubusercontent.com/23195286/164990207-0c608444-dc0c-4f37-8260-05bdc6fb1a7e.png">
<img width="1199" alt="Screenshot 2022-04-24 at 20 04 42" src="https://user-images.githubusercontent.com/23195286/164990213-937d1e35-d655-4411-860b-519eb2515baa.png">

**Comment by pandrr** [comment, marked as answer]
very strange, they look fine on my chrome, win and mac

**Reply by steam0r** [reply]
also looking fine on my mac and windows machines, this might be a hardware/driver issue? you can use https://cables.gl/browser to check on different machines and find differences.

**Reply by davidmaruscsak00** [reply]
Thank you for the replies.  I am going to check the differences.
